packages/react-native-reanimated/android/src/main/cpp/worklets/CMakeLists.txt

```
@@ -118,7 +118,7 @@ elseif(${JS_RUNTIME} STREQUAL "v8")
         PRIVATE
         "${JS_RUNTIME_DIR}/src"
     )
-    file(GLOB V8_SO_DIR "${JS_RUNTIME_DIR}/android/build/intermediates/library_jni/*/jni/${ANDROID_ABI}")
+    file(GLOB V8_SO_DIR "${JS_RUNTIME_DIR}/android/build/intermediates/library_jni/**/*/jni/${ANDROID_ABI}")
```

Isn't it enough here?
```suggestion
    file(GLOB V8_SO_DIR "${JS_RUNTIME_DIR}/android/build/intermediates/library_jni/**/jni/${ANDROID_ABI}")
```